### PR TITLE
Adding Json file for LinkFree

### PR DESCRIPTION
**Pull request by Shivam-Shrivastav**
Added a json file in linkfree

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/411"></a>

### CODE REVIEW

File: public/data/shivamshrivastava.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        "url": "http://github.com/Shivam-Shrivastav",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "http://twitter.com/ShivamsTwt",
+        "icon": "twitter"
+      },
+      {
+        "name": "Learn more about Open Source on my YouTube channel",
```
Comment (CodingSpecies) [inline public/data/shivamshrivastava.json:17]: @Shivam-Shrivastav is this meant to display LinkedIn, not YouTube? Thanks.

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

File: public/data/shivamshrivastava.json (hunk 1 above)
Comment (Shivam-Shrivastav) [reply public/data/shivamshrivastava.json:17]: Okay I am gonna change it to LinkedIn, Thankyou.

**Review by CodingSpecies — APPROVED** [review]
Thank you @Shivam-Shrivastav !

**Review by rohan-kulkarni-25 — APPROVED** [review]
LGTM 💥

**Pull request merged**